src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs

```diff
@@ -247,11 +247,11 @@ private List<string> AddArgs()
                 var quietTestLogging = new List<string>() {"q", "quiet"};
 
                 string vsTestVerbosity = "minimal";
-                if (normalTestLogging.Contains(this.VSTestVerbosity))
+                if (normalTestLogging.Contains(this.VSTestVerbosity.ToLowerInvariant()))
```

Comment:
Were only these 2 affected ?

Reply:
Yes, that were the only two case sensitive checks I could find in that file and falsify with tests. There is also https://github.com/microsoft/vstest/blob/fa629d217b154ca729547d16249738d2ac3e9fbc/src/Microsoft.TestPlatform.Build/Tasks/VSTestTask.cs#L194 but that is unrelated to the issue, and it did not look like it would easily break. We can start another issue if you think this is an improvement that should be made.